File: Exemplos/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/SAT/associar_assinatura.dart
import 'package:intent_digital_hub/IntentDigitalHubService/SAT/sat_command.dart';

class AssociarAssinatura extends SatCommand {
  final int _numSessao;
  final String _codAtivacao;
  final String _cnpjSH;
  final String _assinaturaAC;

  AssociarAssinatura(
      this._numSessao, this._codAtivacao, this._cnpjSH, this._assinaturaAC)
      : super('AssociarAssinatura');

  @override
  functionParameters() {
    return '"numSessao"'
        ':'
        '$_numSessao'
        ','
        '"codAtivacao"'
        ':'
        '"$_codAtivacao"'
        ','
        '"cnpjSH"'
        ':'
        '"$_cnpjSH"'
        ','
        '"assinaturaAC"'
        ':'
        '"$_assinaturaAC"';
  }
}

File: Exemplos/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Java-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/SAT/AssociarAssinatura.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.SAT

import com.elgin.intent_digital_hub.IntentDigitalHubService.SAT.SatCommand

class AssociarAssinatura(
    private val numSessao: Int,
    private val codAtivacao: String,
    private val cnpjSH: String,
    private val assinaturaAC: String
) : SatCommand("AssociarAssinatura") {
    override fun functionParameters(): String {
        return "\"numSessao\"" + ":" + numSessao + "," +
                "\"codAtivacao\"" + ":" + "\"" + codAtivacao + "\"" + "," +
                "\"cnpjSH\"" + ":" + "\"" + cnpjSH + "\"" + "," +
                "\"assinaturaAC\"" + ":" + "\"" + assinaturaAC + "\""
    }
}